prow/cluster/jobs/istio/operator/istio.operator.master.yaml

```diff
@@ -36,6 +36,33 @@ presubmits:
       nodeSelector:
         testing: test-pool
 
+  - name: operator-build-master
+    <<: *job_template
+    context: prow/build.sh
+    always_run: true
+  # TODO: make required once stable
+    optional: true
+    spec:
+      containers:
+      - <<: *istio_container
+        command:
+        - entrypoint
+        - prow/build.sh
+
+  - name: operator-test-master
+    <<: *job_template
+    context: prow/test.sh
+    always_run: true
+  # TODO: make required once stable
+    optional: true
+    spec:
+      containers:
+      - <<: *istio_container
+        command:
+        - entrypoint
+        - prow/build.sh
```

Comment:
is this supposed to be prow/test.sh ?

Reply:
Thanks will resolve when back in the office.